app/views/hiring_staff/vacancies/_new_tag.html.haml

```diff
@@ -0,0 +1,4 @@
+- attributes.each do |attribute|
```

Comment:
The filename isn't obvious enough for my liking. Can we do e.g. _new_attributes_tag?

Reply:
Certainly